CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+### 2.1.0 (unreleased)
+
+New features:
+ * `WhitelistCrowdsale`: a crowdsale where only whitelisted accounts (`WhitelistedRole`) can purchase tokens. This attribute is given by a series of whitelisters (`WhitelisterRole`), who can also remove them. Similar to the pre-2.0 `WhitelistedCrowdsale`.
+ * `RefundablePostDeliveryCrowdsale`: replacement for the (deprecated) `RefundableCrowdsale`: tokens are only granted once the crowdsale ends (if it meets its goal).
+ * `ERC20`: `transferFrom` and `_burnFrom ` now emit `Approval` events, improving dApp support.
+ * `ERC721`: added `_burn(uint256 tokenId)`, replacing the (deprecated) old function.
+ * `ERC721`: added `_tokensOfOwner(address owner)`, allowing for more extensive `ERC721` customization.
+
+Improvements:
+ * All contracts now require solc v0.4.24 or higher.
```

Comment:
It was only one contract, right? I'd be more explicit about that (i.e. what changed exactly) because otherwise users will wonder if this affects them in any way.

Reply:
Yup. I actually added this thinking we'd go up to 0.4.25, but we haven't discussed that :stuck_out_tongue:

Reply:
Heh, yeah. IMO let's wait until next release when we figure out a strategy for dealing with solc updates.